src/webparts/laserficheAdminConfiguration/components/ManageMappingsPage/ManageMappingsPage.tsx

```diff
@@ -678,12 +675,8 @@ export default function ManageMappingsPage(
               <table className='table table-sm'>
                 <thead>
                   <tr>
-                    <th className='text-center' style={{ width: '45%' }}>
```

Comment:
I'm noticing that you're deleting styles that set width, but I don't understand your motivation.